screens/ExpenseTrackerScreen.js

```diff
@@ -0,0 +1,150 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { AsyncStorage, Button, FlatList, StyleSheet, Text, TextInput, View } from 'react-native';
+
+import Colors from '../constants/Colors';
+
+export default function ExpenseTrackerScreen() {
+  const [expenses, setExpenses] = useState([]);
+  const [inputValues, setInputValues] = useState({ title: 'Expense', amount: '100' });
```

Comment:
With setting initial values, there is something written in the input fields. I decided to do this to achieve a placeholder, a text describing the input field, because there is no label. With empty fields, no one would know what to fill in.